test/e2e/run-e2e-test.js

```diff
@@ -117,6 +122,10 @@ async function main() {
   const configFile = path.join(__dirname, '.mocharc.js');
   const extraArgs = process.env.E2E_ARGS?.split(' ') || [];
 
+  if (mmi) {
+    extraArgs.push('-g', '@no-mmi', '-i');
+  }
+
```

Comment:
Could you add inline documentation here for what these flags do (grep/invert) and explain the @no-mmi tag on some of the tests